librz/core/cmd_descs/cmd_print.yaml

```diff
@@ -494,6 +494,45 @@ commands:
         optional: true
     modes:
       - RZ_OUTPUT_MODE_JSON
+  - name: pr
+    summary: Print raw bytes in different representations
+    subcommands:
+      - name: pr
+        summary: Print raw bytes
+        cname: cmd_print_raw
+        args:
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
```

Comment:
I think this should be rznum. You want $S and co to work here. Also use rz-num-math(code->rnum